install_config/topics/configuring_azure_load_balancer.adoc

```diff
@@ -0,0 +1,117 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-load-balancer_{context}']
+= Using the Microsoft Azure external load balancer as a service
+
+{product-title} can leverage the Microsoft Azure load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in Microsoft Azure and creates the proper firewall rules.
+
+[IMPORTANT]
+====
+Currently, a bug causes extra variables to be included in the Microsoft
+Azure cloud provider when enabling external load balancers. See the
+following for more information:
+
+* https://bugzilla.redhat.com/show_bug.cgi?id=1613546
+====
+
+.Prerequisites
+
+The following items must be added to the
+xref:../install_config/configuring_azure.adoc#azure-configuration-file[the Azure configuration file] located at *_/etc/origin/cloudprovider/azure.conf_*.
```

Comment:
This says "the following items", but I'm not sure what this is referring to. Any tips?

Reply:
That was probably a miscommunication on my end. The items https://docs.openshift.com/container-platform/3.10/install_config/configuring_azure.html#azure-configuration-file must be in place